### DISCUSSION TITLE
Updating docs without pushing a release

### DISCUSSION THREAD

**Discussion by cosenal** [unitaryfund/mitiq#2552] [Ideas]
I find it odd and inconvenient that we have to publish a new Mitiq release when what we really want to do is just removing a banner from a website. We don't really want the users to upgrade their package from pypi just because we removed a banner. 

Here is a solution by Simon Willison that achieves exactly what we want instead: https://til.simonwillison.net/readthedocs/stable-docs 

_tl;dr_: maintain a `stable` branch that we can edit manually, but that will also be wiped up at every release and replaced with the content of the release. The docs will be published from that branch instead of the latest tag. All of this handled automatically by GitHub Actions.

@natestemen wdyt?

**Comment by natestemen** [comment]
I agree that it's a somewhat strange process, a bit clunky, and not ideal. I do feel like our release process is light enough where it's not a big deal, however. If we run into this scenario again, lets come back to this.
